src/coreclr/jit/utils.cpp

```diff
@@ -1643,6 +1644,7 @@ void HelperCallProperties::init()
             case CORINFO_HELP_TYPEHANDLE_TO_RUNTIMETYPE:
             case CORINFO_HELP_TYPEHANDLE_TO_RUNTIMETYPEHANDLE:
 
+                // isNoEscape = true;
```

Comment:
Delete?

Reply:
Yeah, for now. I think we can be more ambitious here someday.